src/plugins/orangehrmAdminPlugin/test/fixtures/testcases/ModulesAPITestCases.yaml

```diff
@@ -37,15 +38,16 @@ Update:
       performance: false,
       maintenance: false,
       mobile: false,
-      directory: false
+      directory: false,
+      claim: false
```

Comment:
```suggestion
```
same here. All existing API tests request body should not be changed. Write new test cases for cover claim.